src/solver_interface.jl

```diff
@@ -149,8 +149,25 @@ function get_number_linear_constraints end
 
 Adds linear constraints of the form `Ax (sense) rhs` to the model `m`.
 
-The A matrix is given in triplet form `A[rows[i], cols[i]] = coef[i]` for all
-`i`, and `length(rows) == length(cols) == length(coefs)`.
+The A matrix is given in triplet form `A[row(i), cols[i]] = coef[i]` for all
+`i` where `row(i)` is the largest element in `rows` that is smaller or equal 
+than `i`. `rows` is a list of integers sorted in increasing order. It contains 
+the starting index (of `cols`) for each row. `length(cols) == length(coefs)` 
+and `length(rows) == length(sense) == length(rhs)`.
```

Comment:
We should be moving away from interfaces that have lists of arrays of equal length when possible. Why not use a struct instead? Why are we redefining CSR matrices?

Reply:
Most solvers use CSR Why should they Need an extra conversion layer?

Reply:
What about defining a struct for the CSR matrix then?

Reply:
That would be nice!